docker/setup_wasmgovd.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+#set -o errexit -o nounset -o pipefail
+
+PASSWORD=${PASSWORD:-1234567890}
+STAKE=${STAKE_TOKEN:-ustake}
+FEE=${FEE_TOKEN:-ucosm}
+
+wasmgovd init --chain-id=testing testing
```

Comment:
Do we want to add some default parameters for the permissioning in this demo genesis.

There are controls of who can store code and who can instantiate code (by default). I assume those are Everybody by default (which is great for `wasmd` and should remain so). How about adding a line to the genesis file here to set them both to `Nobody`, thus requiring governance permission for everything?

Reply:
👍  makes sense but I have noticed that the genesis is not really easy to parse. 
```json
  "wasm": {
      "params": {
        "upload_access": {
          "type": 3,
          "address": ""
        },
        "instantiate_default_permission": 3
      },
      "codes": null,
      "contracts": null,
      "sequences": null
    },
```
I would add some changes that we get:
```json
  "wasm": {
      "params": {
        "code_upload_access": {
          "permission": "Everybody",
        },
        "instantiate_default_permission": "Everybody",
      },
    },
```
so that we can replace `permission": "Everybody",` with `permission": "Nobody",`

Reply:
Sounds great. Loving names there not numbers.